dashboard/legacy/middleware/helpers/projects.rb

```diff
@@ -457,4 +464,27 @@ def project_type_from_merged_row(row)
     # Report these as 'unknown'.
     'unknown'
   end
+
+  # Temporarily logging rather than erroring in order
+  # to confirm that only valid thumbnail URLs are present in existing projects.
+  # DCDO flag in place to choose when to enable/disable logging (off by default).
+  def validate_thumbnail_url(channel_id, thumbnail_url)
+    return true unless thumbnail_url
+
+    # valid thumbnail URLs should be of the format:
+    # /v3/files/<channel_id>/.metadata/thumbnail.png
+    # I observed thumbnail URLs of remixed projects having having the channel ID of the parent project,
+    # so we assert on the start/end of the URL
+    valid_thumbnail_url = thumbnail_url.start_with?('/v3/files/') && thumbnail_url.end_with?('.metadata/thumbnail.png')
```

Comment:
somehow, My brains ask me to ask you to create a method for this
```
def is_valid_thumbnail_url(thumbnail_url) 
  thumbnail_url.start_with?('/v3/files/') && thumbnail_url.end_with?('.metadata/thumbnail.png')
end 
```

The reason is that I have seen many times where the logic for something valid changes overtime, and having this logic wrapped is always nice and creates higher confidence when updating

Reply:
Yep that makes sense to me! Will update.